RasimKhusaenov/my_own_blog
# Add ability to update any article by company owner

### Summary

Added ability to update any article by company owner.

### How it works

https://user-images.githubusercontent.com/35406008/221017263-47214049-59ad-45c3-9ca4-577ae07075c7.mov

### Test plan

List of steps to manually test introduced functionality:

- Log in as writer of the company
- Create article
- Log in as owner of the company
- Go to writer's article
- Edit/destroy it
- Make sure everything is OK

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
- Resolves #46 Issue
